src/react-redux.js

```diff
@@ -101,6 +101,13 @@ export function connectGlobal<SP, P, C: ComponentType<P>>(
  * function effectively gets no type-checking of anything it does with it.
  */
 export function useSelector<SS>(
+  selector: (state: PerAccountState) => SS,
+  equalityFn?: (a: SS, b: SS) => boolean,
+): SS {
+  return useSelectorInner<PerAccountState, SS>(selector, equalityFn);
+}
+
+export function useGlobalSelector<SS>(
```

Comment:
Hmm, good catch, thanks!

It looks like what happened in those cases is:
* One was a change written before we had the type-wrapper, and we didn't notice that point when rebasing and merging.
* The other was originally an equally mistaken direct import of `connect`. We didn't notice in code review when merging, and didn't happen to notice later when converting to Hooks.

It'd be nice to have a lint rule against these imports, because it's an easy thing to do by accident (especially for contributors new to our codebase). I'll take a quick look to see if there's a real easy way to do that.

Reply:
#5093